module-sys/Service/Worker.cpp

```diff
@@ -222,12 +220,8 @@ namespace sys
 
     bool Worker::stop()
     {
-        if (runnerTask) {
-            assert(xTaskGetCurrentTaskHandle() == runnerTask);
-            assert(getState() == State::Running);
-            return sendControlMessage(ControlMessage::Stop);
-        }
-        return true;
+        assert(getState() == State::Running);
```

Comment:
asserts don't work on release builds - if you meant to keep the check, I'd suggest switching to `if`

Reply:
It was earlier for testing purposes so I left them except `assert(xTaskGetCurrentTaskHandle() == runnerTask);` that wasn't necessary :)